### PR TITLE
mruby can't parse string literal concatenation

### PR DESCRIPTION
**Pull request by ksss**

### CODE REVIEW

File: core/io/fixtures/classes.rb [hunk 1]
```diff
@@ -71,7 +71,7 @@ def self.lines_space_separator_limit
   end
 
   def self.lines_r_separator
-    [ "Voici la ligne une.\nQui \303\250 la linea due.\n\n\n" \
+    [ "Voici la ligne une.\nQui \303\250 la linea due.\n\n\n" << \
```
Comment (eregon) [inline core/io/fixtures/classes.rb]: I think we could just have the entire string as a single literal.
It's a bit long, but then it's clearer it's just one String and there is no concatenation involved.

**Comment by eregon** [comment]
This looks good.
In general I prefer no concatenation at all (`"line1\nline2"`) or `+`:
```ruby
"line1\n" +
"line2"
```

**Comment by stomar** [comment]
The backslash after the `<<` operator shouldn't be necessary, or should it?

**Comment by ksss** [comment]
@eregon @stomar Thank you for reviewing.
How about a string single literal?

(I found a most longest line in ruby/spec https://github.com/ruby/spec/blob/f96cb7cf9ec8c9f5e38d0e40888ac5b5d895dc3d/library/zlib/crc_table_spec.rb#L8)

**Comment by eregon** [comment]
Thanks!

> I found a most longest line in ruby/spec 

Ahah, I'll fix that one.

**Pull request merged**